diff -durN lbreakout-001014.orig/lbreakout/Makefile.am lbreakout-001014/lbreakout/Makefile.am
--- lbreakout-001014.orig/lbreakout/Makefile.am	Wed Oct 11 20:41:12 2000
+++ lbreakout-001014/lbreakout/Makefile.am	Sun Nov 12 17:47:43 2000
@@ -25,51 +25,51 @@
 manual/x10.jpg
 
 install-data-local:
-	$(mkinstalldirs) $(sdir)
-	$(INSTALL_DATA) ball.bmp $(sdir)/ball.bmp 
-	$(INSTALL_DATA) bkgnd0.bmp $(sdir)/bkgnd0.bmp 
-	$(INSTALL_DATA) bkgnd1.bmp $(sdir)/bkgnd1.bmp 
-	$(INSTALL_DATA) bkgnd2.bmp $(sdir)/bkgnd2.bmp 
-	$(INSTALL_DATA) bricks.bmp $(sdir)/bricks.bmp 
-	$(INSTALL_DATA) club.bmp $(sdir)/club.bmp 
-	$(INSTALL_DATA) extras.bmp $(sdir)/extras.bmp 
-	$(INSTALL_DATA) life.bmp $(sdir)/life.bmp 
-	$(INSTALL_DATA) logo.bmp $(sdir)/logo.bmp 
-	$(INSTALL_DATA) numbers.bmp $(sdir)/numbers.bmp
-	$(INSTALL_DATA) shot.bmp $(sdir)/shot.bmp
-	$(INSTALL_DATA) title.bmp $(sdir)/title.bmp
-	$(INSTALL_DATA) weapon.bmp $(sdir)/weapon.bmp
-	$(INSTALL_DATA) shine2.bmp $(sdir)/shine2.bmp 
-	$(INSTALL_DATA) fr_l.bmp $(sdir)/fr_l.bmp 
-	$(INSTALL_DATA) fr_luc.bmp $(sdir)/fr_luc.bmp 
-	$(INSTALL_DATA) fr_r.bmp $(sdir)/fr_r.bmp 
-	$(INSTALL_DATA) fr_rlc.bmp $(sdir)/fr_rlc.bmp 
-	$(INSTALL_DATA) fr_ruc.bmp $(sdir)/fr_ruc.bmp 
-	$(INSTALL_DATA) fr_t.bmp $(sdir)/fr_t.bmp  
-	$(INSTALL_DATA) f_white.bmp $(sdir)/f_white.bmp
-	$(INSTALL_DATA) f_yellow.bmp $(sdir)/f_yellow.bmp
-	$(INSTALL_DATA) font_s.sdlfnt $(sdir)/font_s.sdlfnt 
-	$(INSTALL_DATA) click.wav $(sdir)/click.wav
-	$(INSTALL_DATA) damn.wav $(sdir)/damn.wav
-	$(INSTALL_DATA) damnit.wav $(sdir)/damnit.wav 
-	$(INSTALL_DATA) excellent.wav $(sdir)/excellent.wav 
-	$(INSTALL_DATA) exp.wav $(sdir)/exp.wav 
-	$(INSTALL_DATA) expand.wav $(sdir)/expand.wav 
-	$(INSTALL_DATA) freeze.wav $(sdir)/freeze.wav 
-	$(INSTALL_DATA) gainlife.wav $(sdir)/gainlife.wav 
-	$(INSTALL_DATA) looselife.wav $(sdir)/looselife.wav 
-	$(INSTALL_DATA) metal.wav $(sdir)/metal.wav 
-	$(INSTALL_DATA) reflect.wav $(sdir)/reflect.wav 
-	$(INSTALL_DATA) score.wav $(sdir)/score.wav 
-	$(INSTALL_DATA) shot.wav $(sdir)/shot.wav 
-	$(INSTALL_DATA) shrink.wav $(sdir)/shrink.wav 
-	$(INSTALL_DATA) slime.wav $(sdir)/slime.wav 
-	$(INSTALL_DATA) verygood.wav $(sdir)/verygood.wav 
-	$(INSTALL_DATA) wall.wav $(sdir)/wall.wav 
-	$(INSTALL_DATA) weapon.wav $(sdir)/weapon.wav 
-	$(INSTALL_DATA) wontgiveup.wav $(sdir)/wontgiveup.wav 
-	$(mkinstalldirs) $(hdir)
-	$(INSTALL_DATA) -m 666 highscore $(hdir)/highscore
+	$(mkinstalldirs) $(DESTDIR)$(sdir)
+	$(INSTALL_DATA) ball.bmp $(DESTDIR)$(sdir)/ball.bmp 
+	$(INSTALL_DATA) bkgnd0.bmp $(DESTDIR)$(sdir)/bkgnd0.bmp 
+	$(INSTALL_DATA) bkgnd1.bmp $(DESTDIR)$(sdir)/bkgnd1.bmp 
+	$(INSTALL_DATA) bkgnd2.bmp $(DESTDIR)$(sdir)/bkgnd2.bmp 
+	$(INSTALL_DATA) bricks.bmp $(DESTDIR)$(sdir)/bricks.bmp 
+	$(INSTALL_DATA) club.bmp $(DESTDIR)$(sdir)/club.bmp 
+	$(INSTALL_DATA) extras.bmp $(DESTDIR)$(sdir)/extras.bmp 
+	$(INSTALL_DATA) life.bmp $(DESTDIR)$(sdir)/life.bmp 
+	$(INSTALL_DATA) logo.bmp $(DESTDIR)$(sdir)/logo.bmp 
+	$(INSTALL_DATA) numbers.bmp $(DESTDIR)$(sdir)/numbers.bmp
+	$(INSTALL_DATA) shot.bmp $(DESTDIR)$(sdir)/shot.bmp
+	$(INSTALL_DATA) title.bmp $(DESTDIR)$(sdir)/title.bmp
+	$(INSTALL_DATA) weapon.bmp $(DESTDIR)$(sdir)/weapon.bmp
+	$(INSTALL_DATA) shine2.bmp $(DESTDIR)$(sdir)/shine2.bmp 
+	$(INSTALL_DATA) fr_l.bmp $(DESTDIR)$(sdir)/fr_l.bmp 
+	$(INSTALL_DATA) fr_luc.bmp $(DESTDIR)$(sdir)/fr_luc.bmp 
+	$(INSTALL_DATA) fr_r.bmp $(DESTDIR)$(sdir)/fr_r.bmp 
+	$(INSTALL_DATA) fr_rlc.bmp $(DESTDIR)$(sdir)/fr_rlc.bmp 
+	$(INSTALL_DATA) fr_ruc.bmp $(DESTDIR)$(sdir)/fr_ruc.bmp 
+	$(INSTALL_DATA) fr_t.bmp $(DESTDIR)$(sdir)/fr_t.bmp  
+	$(INSTALL_DATA) f_white.bmp $(DESTDIR)$(sdir)/f_white.bmp
+	$(INSTALL_DATA) f_yellow.bmp $(DESTDIR)$(sdir)/f_yellow.bmp
+	$(INSTALL_DATA) font_s.sdlfnt $(DESTDIR)$(sdir)/font_s.sdlfnt 
+	$(INSTALL_DATA) click.wav $(DESTDIR)$(sdir)/click.wav
+	$(INSTALL_DATA) damn.wav $(DESTDIR)$(sdir)/damn.wav
+	$(INSTALL_DATA) damnit.wav $(DESTDIR)$(sdir)/damnit.wav 
+	$(INSTALL_DATA) excellent.wav $(DESTDIR)$(sdir)/excellent.wav 
+	$(INSTALL_DATA) exp.wav $(DESTDIR)$(sdir)/exp.wav 
+	$(INSTALL_DATA) expand.wav $(DESTDIR)$(sdir)/expand.wav 
+	$(INSTALL_DATA) freeze.wav $(DESTDIR)$(sdir)/freeze.wav 
+	$(INSTALL_DATA) gainlife.wav $(DESTDIR)$(sdir)/gainlife.wav 
+	$(INSTALL_DATA) looselife.wav $(DESTDIR)$(sdir)/looselife.wav 
+	$(INSTALL_DATA) metal.wav $(DESTDIR)$(sdir)/metal.wav 
+	$(INSTALL_DATA) reflect.wav $(DESTDIR)$(sdir)/reflect.wav 
+	$(INSTALL_DATA) score.wav $(DESTDIR)$(sdir)/score.wav 
+	$(INSTALL_DATA) shot.wav $(DESTDIR)$(sdir)/shot.wav 
+	$(INSTALL_DATA) shrink.wav $(DESTDIR)$(sdir)/shrink.wav 
+	$(INSTALL_DATA) slime.wav $(DESTDIR)$(sdir)/slime.wav 
+	$(INSTALL_DATA) verygood.wav $(DESTDIR)$(sdir)/verygood.wav 
+	$(INSTALL_DATA) wall.wav $(DESTDIR)$(sdir)/wall.wav 
+	$(INSTALL_DATA) weapon.wav $(DESTDIR)$(sdir)/weapon.wav 
+	$(INSTALL_DATA) wontgiveup.wav $(DESTDIR)$(sdir)/wontgiveup.wav 
+	$(mkinstalldirs) $(DESTDIR)$(hdir)
+	$(INSTALL_DATA) -m 666 highscore $(DESTDIR)$(hdir)/highscore
 
 uninstall-local:
 	rm -fr $(sdir)
diff -durN lbreakout-001014.orig/lbreakout/Makefile.in lbreakout-001014/lbreakout/Makefile.in
--- lbreakout-001014.orig/lbreakout/Makefile.in	Sat Oct 14 14:10:48 2000
+++ lbreakout-001014/lbreakout/Makefile.in	Sun Nov 12 17:47:08 2000
@@ -308,51 +308,51 @@
 
 
 install-data-local:
-	$(mkinstalldirs) $(sdir)
-	$(INSTALL_DATA) ball.bmp $(sdir)/ball.bmp 
-	$(INSTALL_DATA) bkgnd0.bmp $(sdir)/bkgnd0.bmp 
-	$(INSTALL_DATA) bkgnd1.bmp $(sdir)/bkgnd1.bmp 
-	$(INSTALL_DATA) bkgnd2.bmp $(sdir)/bkgnd2.bmp 
-	$(INSTALL_DATA) bricks.bmp $(sdir)/bricks.bmp 
-	$(INSTALL_DATA) club.bmp $(sdir)/club.bmp 
-	$(INSTALL_DATA) extras.bmp $(sdir)/extras.bmp 
-	$(INSTALL_DATA) life.bmp $(sdir)/life.bmp 
-	$(INSTALL_DATA) logo.bmp $(sdir)/logo.bmp 
-	$(INSTALL_DATA) numbers.bmp $(sdir)/numbers.bmp
-	$(INSTALL_DATA) shot.bmp $(sdir)/shot.bmp
-	$(INSTALL_DATA) title.bmp $(sdir)/title.bmp
-	$(INSTALL_DATA) weapon.bmp $(sdir)/weapon.bmp
-	$(INSTALL_DATA) shine2.bmp $(sdir)/shine2.bmp 
-	$(INSTALL_DATA) fr_l.bmp $(sdir)/fr_l.bmp 
-	$(INSTALL_DATA) fr_luc.bmp $(sdir)/fr_luc.bmp 
-	$(INSTALL_DATA) fr_r.bmp $(sdir)/fr_r.bmp 
-	$(INSTALL_DATA) fr_rlc.bmp $(sdir)/fr_rlc.bmp 
-	$(INSTALL_DATA) fr_ruc.bmp $(sdir)/fr_ruc.bmp 
-	$(INSTALL_DATA) fr_t.bmp $(sdir)/fr_t.bmp  
-	$(INSTALL_DATA) f_white.bmp $(sdir)/f_white.bmp
-	$(INSTALL_DATA) f_yellow.bmp $(sdir)/f_yellow.bmp
-	$(INSTALL_DATA) font_s.sdlfnt $(sdir)/font_s.sdlfnt 
-	$(INSTALL_DATA) click.wav $(sdir)/click.wav
-	$(INSTALL_DATA) damn.wav $(sdir)/damn.wav
-	$(INSTALL_DATA) damnit.wav $(sdir)/damnit.wav 
-	$(INSTALL_DATA) excellent.wav $(sdir)/excellent.wav 
-	$(INSTALL_DATA) exp.wav $(sdir)/exp.wav 
-	$(INSTALL_DATA) expand.wav $(sdir)/expand.wav 
-	$(INSTALL_DATA) freeze.wav $(sdir)/freeze.wav 
-	$(INSTALL_DATA) gainlife.wav $(sdir)/gainlife.wav 
-	$(INSTALL_DATA) looselife.wav $(sdir)/looselife.wav 
-	$(INSTALL_DATA) metal.wav $(sdir)/metal.wav 
-	$(INSTALL_DATA) reflect.wav $(sdir)/reflect.wav 
-	$(INSTALL_DATA) score.wav $(sdir)/score.wav 
-	$(INSTALL_DATA) shot.wav $(sdir)/shot.wav 
-	$(INSTALL_DATA) shrink.wav $(sdir)/shrink.wav 
-	$(INSTALL_DATA) slime.wav $(sdir)/slime.wav 
-	$(INSTALL_DATA) verygood.wav $(sdir)/verygood.wav 
-	$(INSTALL_DATA) wall.wav $(sdir)/wall.wav 
-	$(INSTALL_DATA) weapon.wav $(sdir)/weapon.wav 
-	$(INSTALL_DATA) wontgiveup.wav $(sdir)/wontgiveup.wav 
-	$(mkinstalldirs) $(hdir)
-	$(INSTALL_DATA) -m 666 highscore $(hdir)/highscore
+	$(mkinstalldirs) $(DESTDIR)$(sdir)
+	$(INSTALL_DATA) ball.bmp $(DESTDIR)$(sdir)/ball.bmp 
+	$(INSTALL_DATA) bkgnd0.bmp $(DESTDIR)$(sdir)/bkgnd0.bmp 
+	$(INSTALL_DATA) bkgnd1.bmp $(DESTDIR)$(sdir)/bkgnd1.bmp 
+	$(INSTALL_DATA) bkgnd2.bmp $(DESTDIR)$(sdir)/bkgnd2.bmp 
+	$(INSTALL_DATA) bricks.bmp $(DESTDIR)$(sdir)/bricks.bmp 
+	$(INSTALL_DATA) club.bmp $(DESTDIR)$(sdir)/club.bmp 
+	$(INSTALL_DATA) extras.bmp $(DESTDIR)$(sdir)/extras.bmp 
+	$(INSTALL_DATA) life.bmp $(DESTDIR)$(sdir)/life.bmp 
+	$(INSTALL_DATA) logo.bmp $(DESTDIR)$(sdir)/logo.bmp 
+	$(INSTALL_DATA) numbers.bmp $(DESTDIR)$(sdir)/numbers.bmp
+	$(INSTALL_DATA) shot.bmp $(DESTDIR)$(sdir)/shot.bmp
+	$(INSTALL_DATA) title.bmp $(DESTDIR)$(sdir)/title.bmp
+	$(INSTALL_DATA) weapon.bmp $(DESTDIR)$(sdir)/weapon.bmp
+	$(INSTALL_DATA) shine2.bmp $(DESTDIR)$(sdir)/shine2.bmp 
+	$(INSTALL_DATA) fr_l.bmp $(DESTDIR)$(sdir)/fr_l.bmp 
+	$(INSTALL_DATA) fr_luc.bmp $(DESTDIR)$(sdir)/fr_luc.bmp 
+	$(INSTALL_DATA) fr_r.bmp $(DESTDIR)$(sdir)/fr_r.bmp 
+	$(INSTALL_DATA) fr_rlc.bmp $(DESTDIR)$(sdir)/fr_rlc.bmp 
+	$(INSTALL_DATA) fr_ruc.bmp $(DESTDIR)$(sdir)/fr_ruc.bmp 
+	$(INSTALL_DATA) fr_t.bmp $(DESTDIR)$(sdir)/fr_t.bmp  
+	$(INSTALL_DATA) f_white.bmp $(DESTDIR)$(sdir)/f_white.bmp
+	$(INSTALL_DATA) f_yellow.bmp $(DESTDIR)$(sdir)/f_yellow.bmp
+	$(INSTALL_DATA) font_s.sdlfnt $(DESTDIR)$(sdir)/font_s.sdlfnt 
+	$(INSTALL_DATA) click.wav $(DESTDIR)$(sdir)/click.wav
+	$(INSTALL_DATA) damn.wav $(DESTDIR)$(sdir)/damn.wav
+	$(INSTALL_DATA) damnit.wav $(DESTDIR)$(sdir)/damnit.wav 
+	$(INSTALL_DATA) excellent.wav $(DESTDIR)$(sdir)/excellent.wav 
+	$(INSTALL_DATA) exp.wav $(DESTDIR)$(sdir)/exp.wav 
+	$(INSTALL_DATA) expand.wav $(DESTDIR)$(sdir)/expand.wav 
+	$(INSTALL_DATA) freeze.wav $(DESTDIR)$(sdir)/freeze.wav 
+	$(INSTALL_DATA) gainlife.wav $(DESTDIR)$(sdir)/gainlife.wav 
+	$(INSTALL_DATA) looselife.wav $(DESTDIR)$(sdir)/looselife.wav 
+	$(INSTALL_DATA) metal.wav $(DESTDIR)$(sdir)/metal.wav 
+	$(INSTALL_DATA) reflect.wav $(DESTDIR)$(sdir)/reflect.wav 
+	$(INSTALL_DATA) score.wav $(DESTDIR)$(sdir)/score.wav 
+	$(INSTALL_DATA) shot.wav $(DESTDIR)$(sdir)/shot.wav 
+	$(INSTALL_DATA) shrink.wav $(DESTDIR)$(sdir)/shrink.wav 
+	$(INSTALL_DATA) slime.wav $(DESTDIR)$(sdir)/slime.wav 
+	$(INSTALL_DATA) verygood.wav $(DESTDIR)$(sdir)/verygood.wav 
+	$(INSTALL_DATA) wall.wav $(DESTDIR)$(sdir)/wall.wav 
+	$(INSTALL_DATA) weapon.wav $(DESTDIR)$(sdir)/weapon.wav 
+	$(INSTALL_DATA) wontgiveup.wav $(DESTDIR)$(sdir)/wontgiveup.wav 
+	$(mkinstalldirs) $(DESTDIR)$(hdir)
+	$(INSTALL_DATA) -m 666 highscore $(DESTDIR)$(hdir)/highscore
 
 uninstall-local:
 	rm -fr $(sdir)
